src/Spice86.Core/Emulator/Devices/Sound/SoftwareMixer.cs

```diff
@@ -42,7 +42,7 @@ public sealed class SoftwareMixer : IDisposable, IDebuggableComponent {
         float finalVolumeFactor = volumeFactor * (1 + separation);
         
         Span<float> target = stackalloc float[data.Length];
-        for (int i = 0; i < data.Length; i++) {
+        for (int i = 0; i < data.Length; i += 2) {
```

Comment:
Are you sure you only want to process the _even_ entries in `data`?

Reply:
Fixed, the oversight was bigger than I understood on first read.

This was a leftover from when the code was using the imported AudioFrame struct from DOSBox Staging.